lib/config/fileconf.go

```diff
@@ -924,7 +924,11 @@ type AWSKMS struct {
 	// MultiRegion contains configuration for multi-region AWS KMS.
 	MultiRegion struct {
 		// Enabled configures new keys to be multi-region.
-		Enabled bool
+		Enabled bool `yaml:"enabled"`
+		// PrimaryRegion is the region the primary key is located.
+		PrimaryRegion string `yaml:"primary_region"`
+		// ReplicaRegions is a list of regions keys will be replicated to.
+		ReplicaRegions []string `yaml:"replica_regions"`
 	} `yaml:"multi_region,omitempty"`
```

Comment:
might be nicer for users of this type and the equivalent in servicecfg if we gave the MultiRegion struct type a name by defining it outside of the AWSKMS struct